randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1573,14 +1599,21 @@ Extra - map_name: rm_a5b15
   * Layers: default
   * Open Passage to Facility Storage Tower East/Dock to Spiky Space Jump Trial Entrance
   > Dock to Spiky Space Jump Trial
-      Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
+      All of the following:
+          Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
+          Any of the following:
+              # Climb Requirements
+              Hi-Jump Boots or Power Grip or Space Jump or Walljump (Beginner)
```

Comment:
Not a fan of the duplicated info here (which is also missing stuff). Maybe have a new "Top" node to help with things? and then have:
- right exit -> bottom
- left exit -> top
- top -> bottom
- top -> right exit
- bottom -> top